src/actions/schemes.test.js

```diff
@@ -6,7 +6,7 @@ import nock from 'nock';
 
 import * as Actions from 'actions/schemes';
 import {Client4} from 'client';
-import {RequestStatus} from 'constants';
+import {RequestStatus} from '../constants';
```

Comment:
Do you know why we have to import `constants` with a relative path now?

Reply:
Yea, this was unfortunate, but 'constants' comes from Typescript itself: node_modules/@types/node/constants.d.ts